doc/source/user_guide/ug_case_settings.rst

```diff
@@ -359,8 +359,9 @@ forcing_nml
    "", "``mm_per_sec``", "(same as MKS units)", ""
    "", "``m_per_sec``", "", ""
    "``restore_ocn``", "logical", "restore sst to data", "``.false.``"
-   "``tfrz_option``", "``linear_salt``", "linear functino of salinity (ktherm=1)", "``mushy``"
+   "``tfrz_option``", "``linear_salt``", "linear function of salinity (ktherm=1)", "``mushy``"
```

Comment:
This is where the default should be defined and it is.  Could you reorder the options into alphabetical order.  I know it's a pain, but that means constant would be first.  Thanks!